buildtest/buildsystem/base.py

```diff
@@ -319,8 +351,7 @@ def get_environment(self):
 
         env = []
         pairs = self.recipe.get("env")
-        shell = self.get_shell()
-
+        shell = self.shell_name
```

Comment:
This would use the self.shell property.

Reply:
I dont feel comfortable making this change, i think this relates to discussion  https://github.com/buildtesters/buildtest/pull/290#discussion_r417504156

Reply:
Yep happy to review this again in another PR.